sxt/base/macro/cuda_warning.h

```diff
@@ -16,7 +16,8 @@
  */
 #pragma once
 
-#ifdef __CUDACC__
+// #ifdef __CUDACC__
```

Comment:
Nit: do you intend to remove this comment?

Reply:
Fixed .